src/igvfd/upgrade/sample.py

```diff
@@ -234,3 +234,14 @@ def sample_12_13(value, system):
     # default value for 'virtual' property.
     # The default value will be automatically populated.
     return
+
+
+@upgrade_step('multiplexed_sample', '1', '2')
+def sample_13_14(value, system):
```

Comment:
i think we have to capture the popped values in "notes"

Reply:
Got it, I updated the upgrade